### ISSUE TITLE
Create symlinks instead of junctions on Windows?

### ISSUE THREAD

**Issue by hoegge** [timoxley/linklocal#36]
On windows, the junctions created by linklocal means that git clean does not only delete the symlinks (junctions) but also the files in the linked folders, which then prevens the use of git clean with linklocal / yarn. This is apparently not the case on UNIX.

Would it be possible to change linklocal to create symlinks on Windows instead (which does exist in the NTFS filesystem now) (see mklink documentation)?

As documented here [https://www.bountysource.com/issues/990899-git-clean-d-deletes-files-in-junctioned-directories-on-windows](url) `git clean -df` will delete files in folders linked by junctions. I have tried the same thing using `mlink /D link target` instead, and then files in the linked folder is not deleted.

Thanks

**Comment by timoxley** [comment]
I have never used node on windows, if you can submit a pull request that adds a flag for toggling symlinks/junctions i would accept that

**Comment by hoegge** [comment]
Ok. Requires Node 8.10.0 or higher to work (libuv 1.19.1 needed for symlinks to work on Windows). Will come back with pull request

**Comment by timoxley** [comment]
@hoegge is this still an issue?

**Comment by hoegge** [comment]
Yes - I believe so. I considered looking into changing it myself, but didn’t have the time. So unless someone else have fixed it I think it would be - can test at some point.
